pkg/kf/commands/apps/scale.go

```diff
@@ -0,0 +1,125 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package apps
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/google/kf/pkg/apis/kf/v1alpha1"
+	"github.com/google/kf/pkg/kf/commands/config"
+	"github.com/google/kf/pkg/kf/commands/utils"
+	"github.com/google/kf/pkg/kf/kfapps"
+	"github.com/spf13/cobra"
+)
+
+// NewScaleCommand creates a command capable of scaling an app.
+func NewScaleCommand(
+	p *config.KfParams,
+	client kfapps.Client,
+) *cobra.Command {
+	var (
+		instances    int
+		autoscaleMin int
+		autoscaleMax int
+	)
+
+	var scale = &cobra.Command{
+		Use:   "scale APP_NAME",
+		Short: "Change the instance count for an app.",
+		Example: `
+  kf scale myapp --i 3 # Scale to exactly 3 instances
+  kf scale myapp --instances 3 # Scale to exactly 3 instances
+  kf scale myapp --autoscale-min 3 # Autoscaler won't scale below 3 instances
+  kf scale myapp --autoscale-max 5 # Autoscaler won't scale above 5 instances
+  kf scale myapp --autoscale-min 3 --autoscale-max 5 # Autoscaler won't below 3 or above 5 instances
+  `,
+		Args: cobra.ExactArgs(1),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			if err := utils.ValidateNamespace(p); err != nil {
+				return err
+			}
+
+			if instances < 0 && autoscaleMin < 0 && autoscaleMax < 0 {
+				return errors.New("--instances, --autoscaleMin, or --autoscaleMax flag are required")
```

Comment:
nit: flag names here should be `--autoscale-min` and `--autoscale-max`

Reply:
done